### ISSUE TITLE
fix: 🤔 Slotted content is not accessible for sd-tooltip

### ISSUE THREAD

**Issue by Vahid1919** [solid-design-system/solid#1700]
## Current behavior
When replacing the default trigger for sd-tooltip, the content of the tooltip will not be read out loud by screen readers. 

## Expected behavior
The screen reader gets information that is in the tooltip.

## Steps to reproduce

Navigate to the button using a screen reader of your choice: 
https://codepen.io/Vahid-Nesro/pen/qEWOrJb

## Technical Information
We use aria-describedby to connect the tooltip information to the button. But when slotting a custom button, the connect is lost since id relations do not cross to the ShadowRoot. There does not seem to be a concrete solution at the moment (https://www.matuzo.at/blog/2023/web-components-accessibility-faq/aria-references/), but this needs to be solved as part of the solution. 

## Solution

We will create an example in the docs whih documents the feature of a different trigger. In this example we will write a note that describes the accessibility issue we have. And we provide information how to solve this. In our example we will provide a solution with e. g. hidden text or similar or a "details summary" which is only available to screenreaders, whil the tooltip gets an aria-disabled to be not focusable or similar. Good luck!

When working on this, consider fixing #1744 

Solution could be: 

## DoR
- [ ] Item has business value
- [ ] Item has been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch
